cmd/gtfTools/filter.go

```diff
@@ -73,6 +77,13 @@ func gtfFilter(s FilterSettings) {
 			if _, foundInMap = geneNameMap[records[currGene].GeneName]; !foundInMap {
 				pass = false
 			}
+		} else if s.ChromFilter != "" {
```

Comment:
I haven't thought deeply about this, but do you think it makes the most sense to have the GeneNameList override the ChromFilter as it currently does? I suppose if you are asking for a particular gene you probably want to have it in the output even if you only asked for records specifically from a different chromosome, but I think it is less intuitive than applying both filters simultaneously. I could go either way.

Reply:
I hadn't thought about applying them together to be honest, but I can work that out for sure.